demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -457,6 +457,52 @@ def test_bump_version_no_version(self):
         update_rn.metadata_path = os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json')
         self.assertRaises(ValueError, update_rn.bump_version_number)
 
+    def test_add_fromversion_to_new_file(self):
```

Comment:
let's add a "sad" test case - that for a NOT `is_new_file` the `available from version...` line is not generated.

Reply:
done